src/Main.elm

```diff
@@ -11,145 +11,164 @@ import List
-      let updateEntry e =
-        if e.id == id then { e | taps = e.taps + 1 } else e
-      in
-        { model | entries = List.map updateEntry model.entries }
+    case msg of
+        NoOp ->
+            model
+
+        UpdateInput str ->
+            { model | input = str }
+
+        Add ->
+            { model
+                | input = ""
+                , uid = model.uid + 1
+                , entries =
+                    if String.isEmpty model.input then
+                        model.entries
+                    else
+                        model.entries ++ [ newEntry model.input model.uid ]
+            }
+
+        Remove id ->
+            let
+                isNotThis e =
+                    e.id /= id
+            in
+                { model | entries = List.filter isNotThis model.entries }
+
+        Tap id ->
+            let
+                updateEntry e =
+                    if e.id == id then
+                        { e | taps = e.taps + 1 }
+                    else
+                        e
+            in
+                { model | entries = List.map updateEntry model.entries }
 
 
 newEntry : String -> Int -> Entry
 newEntry desc id' =
-  { description = desc
-  , id = id'
-  , taps = 0
-  }
+    { description = desc
+    , id = id'
+    , taps = 0
+    }
 
 
 onEnter : Msg -> Attribute Msg
 onEnter msg =
-  let
-    tagger code =
-      if code == 13 then msg else NoOp
-  in
-    on "keydown" (Json.map tagger keyCode)
+    let
+        tagger code =
+            if code == 13 then
+                msg
+            else
+                NoOp
+    in
+        on "keydown" (Json.map tagger keyCode)
+
 
 
 -- View
 
+
 view : Model -> Html Msg
 view model =
-  div
-    []
-    [ header
+    div
         []
-        [ h1 [] [ text "PRIORITIES" ]
-        , div
-          [ class "input-container" ]
-          [ input
-              [ placeholder "What's your idea?"
-              , autofocus True
-              , value model.input
-              , onInput UpdateInput
-              , onEnter Add
-              ]
-              []
-          ]
-        ]
-    , main'
-        []
-        [ div
-            [ id "todos" ] <|
-            List.map viewEntry model.entries
-        , div
-            [ class "instructions" ]
-            [ p [] [ text "Tap to prioritize." ]
+        [ header
+            []
+            [ h1 [] [ text "PRIORITIES" ]
+            , div
+                [ class "input-container" ]
+                [ input
+                    [ placeholder "What's your idea?"
+                    , autofocus True
+                    , value model.input
+                    , onInput UpdateInput
+                    , onEnter Add
+                    ]
+                    []
+                ]
+            ]
+        , main'
+            []
+            [ div
+                [ id "todos" ]
+              <|
+                List.map viewEntry (List.reverse (List.sortBy .taps model.entries))
```

Comment:
You can change this to:

    List.map viewEntry <| List.reverse <| List.sortBy .taps model.entries

Or if you want to read it from left to right, with data flowing through successive functions:

    List.sortBy .taps model.entries |> List.reverse |> List.map viewEntry